### DISCUSSION TITLE
Cypress hold on to stale DOM reference?

### DISCUSSION THREAD

**Discussion by pckhoi** [cypress-io/cypress#15122] [Questions and Help] [answered]
In the middle of a test Cypress suddenly unable to read node text, or click on node (saying the node is detached). Even though everything is still there. Using `cy.wrap(Cypress.$(` instead of `cy.get(` remedy this for a while until I `cy.visit` another page further down, then it stop working altogether. I'm guessing Cypress was querying off of a stale DOM reference?

I'm using Cypress v6.5.0. I have written many tests before and this is my first time seeing this. Unfortunately my test is long and hard to reproduce and I'm not sure what exactly trigger it.

How should I find clue about what's wrong? Is there a way to force Cypress to reconnect to the DOM?

**Comment by bahmutov** [comment]
Can you at least provide a screenshot and a few test commands around it? 

I would read https://www.cypress.io/blog/2020/07/22/do-not-get-too-detached/ to understand this problem

**Reply by pckhoi** [reply]
I found out the issue. I never close the `.within` brackets. This is embarrassing. Thanks for the assistance!

**Comment by pckhoi** [comment, marked as answer]
I forgot to close the `.within` brackets.
